internal/util/function/bedrock_embedding_provider.go

```diff
@@ -79,12 +80,32 @@ func createBedRockEmbeddingClient(awsAccessKeyId string, awsSecretAccessKey stri
 	return bedrockruntime.NewFromConfig(cfg), nil
 }
 
-func NewBedrockEmbeddingProvider(fieldSchema *schemapb.FieldSchema, functionSchema *schemapb.FunctionSchema, c BedrockClient) (*BedrockEmbeddingProvider, error) {
+func parseAccessInfo(params []*commonpb.KeyValuePair, confParams map[string]string) (string, string) {
```

Comment:
Just check the logic we have right now
1. check `enableConfigAKAndURL` from env
2. get `apikey` and `url` from function param directly
3. check `apikey`, if empty get it from env
4. check `url`, if empty get it from hard code

And we add yml check before these. There are two problems:
1. too complicated 
2. no unified place to these param definition.

Lets do this: 
1. remove all `enable`  params
2. define 3 layer params, function param > env > yml
3. fine a unified place to do this.